rest_framework/fields.py

```diff
@@ -1204,6 +1204,8 @@ def to_representation(self, value):
         )
 
         if output_format.lower() == ISO_8601:
+            if (isinstance(value, str)):
```

Comment:
@areski The `DateTime` was out of this PR scope and now conflict with master. Could you merge it please ?